warehouse/packaging/tasks.py

```diff
@@ -111,3 +116,108 @@ def update_description_html(request):
     for description in descriptions:
         description.html = readme.render(description.raw, description.content_type)
         description.rendered_by = renderer_version
+
+
+@tasks.task(
+    bind=True, ignore_result=True, acks_late=True, retry_backoff=15, retry_jitter=False
+)
+def update_distribution_database(task, request, file, form):
+    """
+    Updates release metadata to public BigQuery database
+    """
+    form_attrs = vars(form) if "__dict__" in form else form
+    file_attrs = vars(file) if "__dict__" in file else file
+    release = file_attrs["release"]
+    release_attrs = vars(release) if "__dict__" in release else release
+
+    bq = request.find_service(name="gcloud.bigquery")
+
+    # Using the schema to populate the data allows us to automatically
+    # set the values rather than assigning values individually
+    def populate_data_using_schema(schema, row_data):
+
+        for sch in schema:
+            if sch.field_type == "RECORD":
+                row_data[sch.name] = dict()
+                populate_data_using_schema(sch.fields, row_data[sch.name])
```

Comment:
If a `RECORD` type field is set to the `REPEATED` mode, they need to be enclosed in an array

Reply:
This is now resolved as the `schema` change invalidated this bug